### PR TITLE
Feature/migrate tick overview endpoint

### PR DESCRIPTION
**Pull request by ahmed-tarek-salem**
**Changes**
- [x] Migrate Explorer info to consume `v1/latest-stats` implemented from 
#105
- [x] Migrate Explorer ticks to consume `v2/epochs/$epoch/ticks` with pagination
- [x] Handle reversed pagination properly
- [x] Handle loading, error, and refresh states properly
- [x] Use sliver builders for better performance

### CODE REVIEW

File: lib/dtos/network_overview_dto.dart [hunk 1]
```diff
@@ -1,59 +1,24 @@
...
-        other.marketCap);
-  }
-
-  factory NetworkOverviewDto.fromJson(Map<String, dynamic> data,
+  factory NetworkTicksDto.fromJson(Map<String, dynamic> json,
       {bool descendingTicks = false}) {
-    List<TickDto> a =
-        data['ticks'].map<TickDto>((e) => TickDto.fromJson(e)).toList();
+    // List<TickDto> a =
+    //     json['ticks'].map<TickDto>((e) => TickDto.fromJson(e)).toList();
```
Comment (sallymoc) [inline lib/dtos/network_overview_dto.dart:17]: if the code is not needed, please remove

File: lib/dtos/network_overview_dto.dart [hunk 2]
```diff
@@ -1,59 +1,24 @@
...
-        data['numberOfEntities'],
-        data['supply'],
-        descendingTicks ? a.reversed.toList() : a,
-        data['price'],
-        data['marketCapitalization']);
+    return NetworkTicksDto(
+      pagination: PaginationResponseModel.fromJson(json['pagination']),
+      ticks: List<TickDto>.from(json['ticks'].map((x) => TickDto.fromJson(x))),
+      //ticks: descendingTicks ? a.reversed.toList() : a,
+    );
```
Comment (sallymoc) [inline lib/dtos/network_overview_dto.dart:22]: if the code is not needed, please remove

File: lib/dtos/network_overview_dto.dart [hunk 3]
```diff
@@ -1,59 +1,24 @@
...
 
-    return NetworkOverviewDto(
-        data['numberOfTicks'],
-        data['numberOfEmptyTicks'],
-        data['numberOfEntities'],
-        data['supply'],
-        descendingTicks ? a.reversed.toList() : a,
-        data['price'],
-        data['marketCapitalization']);
+    return NetworkTicksDto(
```
Comment (sallymoc) [inline lib/dtos/network_overview_dto.dart:18]: if the code above not needed, please remove

**Comment by sallymoc** [comment]
@ahmed-tarek-salem  Tick quality was displaying before 2 decimals and now I see is displaying 4

**Comment by ahmed-tarek-salem** [comment]
@sallymoc 

Resolved, please check.

**Pull request merged**